shared/common-adapters/text.desktop.js

```diff
@@ -53,14 +53,20 @@ export default class Text extends Component {
       'TerminalUsername': {...styles.textTerminal, color: globalColorsDZ2.orange},
       'TerminalPublic': {...styles.textTerminal, color: globalColorsDZ2.yellowGreen2},
       'TerminalPrivate': {...styles.textTerminal, color: globalColorsDZ2.darkBlue2},
-      'TerminalEmpty': {...styles.textTerminal, ...styles.textTerminalEmpty}
+      'TerminalEmpty': {...styles.textTerminal, ...styles.textTerminalEmpty},
+      'InputHeader': styles.textInputHeader
     }[this.props.type]
 
     const opacity = this.props.darkMode ? 1 : 0.75
 
     const color = this.props.darkMode ? globalColorsDZ2.white : globalColorsDZ2.black
 
-    const inlineStyle = this.props.inline ? this._inlineStyle(this.props.type) : {}
+    let inline = true
```

Comment:
The breaks the non inline styling of the terminal components, look at the stylesheet:

<img width="413" alt="screen shot 2016-02-18 at 3 05 14 pm" src="https://cloud.githubusercontent.com/assets/594035/13161200/10f7f2fa-d651-11e5-8978-e59cfbd677ed.png">